src/modules/scraper/service.ts

```diff
@@ -28,138 +28,97 @@ export class ScraperService {
   }
 
   public async run() {
+    this.publishBlocks(ChainIds.mainnet, 10);
+    this.publishBlocks(ChainIds.arbitrum, 4);
+    this.publishBlocks(ChainIds.optimism, 4);
+    this.publishBlocks(ChainIds.polygon, 6);
+    this.publishMerkleDistributorBlocks(30);
+  }
+
+  public async publishBlocks(chainId: number, interval: number) {
     while (true) {
       try {
-        const latestBlocks = await this.getLatestBlocks();
-
-        if (this.appConfig.values.enableSpokePoolsEventsProcessing) {
-          await this.publishBlocks(latestBlocks);
-        }
-        if (this.appConfig.values.enableMerkleDistributorEventsProcessing) {
-          await this.publishMerkleDistributorBlocks(latestBlocks);
```

Comment:
We don't need to handle these conditions anymore? `this.appConfig.values.enableSpokePoolsEventsProcessing` and `this.appConfig.values.enableMerkleDistributorEventsProcessing`?

Reply:
Good point 👍